types/params.go

```diff
@@ -131,6 +142,15 @@ func DefaultABCIParams() ABCIParams {
 	}
 }
 
+func DefaultSynchronyParams() SynchronyParams {
+	// TODO(@wbanfield): Determine experimental values for these defaults
```

Comment:
@sergio-mena can we put together a QA doc and include this

Reply:
@glnro I agree, but some work was already done as part of tendermint/tendermint#7202.
I've just added a reference to tendermint/tendermint#7202 in the task forward-porting default values for the synchrony params, so we can tackle it as part of that task.